.github/workflows/ci-counter.yml

```diff
@@ -0,0 +1,38 @@
+name: NotImplemented
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get Count
+      run: |
+        total_count=$(find rust -type f -exec cat {} + | grep -c NotImplemented)
```

Comment:
I tried this out locally, and I think this has some issues with searching in object files, counting EvalError::NotImplemented and counting the enum definition. I came up with `find rust -iname '*.rs' -type f -exec cat {} + | grep -c -E "(Emit|Parse)Error::NotImplemented"` which is a bit more accurate, but doesn't handle cases like `EmitError::NotImplemented(message) => write!(f, "not implemented: {}", message),`

Reply:
thanks for the tip! i am a little confused about why it doesn't include EmitError::NotImplemented(message) => write!(f, "not implemented: {}", message), -- i think I am missing something, why would it not capture that as well?

Reply:
Ok maybe that was a bit unclear. This [instance](https://github.com/mozilla-spidermonkey/jsparagus/blob/38ecd91f62e5b397d5335a59b5040c1a7f257ae5/rust/emitter/src/emitter.rs#L95) should not be counted, because it's just a way to format the NotImplemented enum and not a real unimplemented feature. However, I think over counting a little bit really isn't a significant problem at the moment.